debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/RecordsStreamProducerIT.java

```diff
@@ -1893,12 +1898,62 @@ public void shouldGracefullySkipTruncateMessages() throws Exception {
         assertEquals(TestHelper.topicName("public.test_table"), record.topic());
         VerifyRecord.isValidInsert(record, PK_FIELD, 2);
 
-        consumer.expects(0);
-        TestHelper.execute("TRUNCATE TABLE public.test_table;");
+        consumer.expects(1);
+        TestHelper.execute("TRUNCATE TABLE public.test_table RESTART IDENTITY CASCADE;");
```

Comment:
What does "RESTART IDENTITY" mean?

Reply:
[`RESTART IDENTITY`](https://www.postgresql.org/docs/11/sql-truncate.html) will restart the sequences of table `public.test_table`. Anyhow, `RESTART IDENTITY / CONTINUE IDENTITY / CASCADE / RESTRICT` is not exposed in the cdc message (result of `PgOutputMessageDecoder.getTruncateOptions` is ignored / unused). It's just there for testing purpose.

Reply:
Ok, thx for clarifying.